### ISSUE TITLE
When is no report

### ISSUE THREAD

**Issue by sysmat** [dependency-check/dependency-check-sonar-plugin#187]
**Describe the bug**

**To Reproduce**
It would be helpful when you support us with [example](https://github.com/dependency-check/dependency-check-sonar-plugin/tree/master/examples) projects.
1. Install '...'
2. Run '....'
3. Login to '....'

**Current behavior**
In sonar when go to dependency-check and there is no report you have javascript error

```
Uncaught (in promise) TypeError: Cannot read property 'value' of undefined
    at report.js:38

```
**Expected behavior**
When is no report you should put some message to user there is no report

**Screenshots**

**Versions (please complete the following information):**

-   dependency-check: 5.2.4
-   sonarqube: 8
-   dependency-check-sonar-plugin: 1.2.6

**Additional context**

**Comment by Reamer** [comment]
Hi @sysmat,
can you check, if the [pre-release](https://github.com/dependency-check/dependency-check-sonar-plugin/releases/tag/2.0.3-SNAPSHOT) solves your issue. This pre-release is based on [rewrite_javascript branch](https://github.com/dependency-check/dependency-check-sonar-plugin/tree/rewrite_javascript).

**Comment by Reamer** [comment]
Fixed by #211
